explainer.md

```diff
@@ -5,6 +5,7 @@
 *   Marijn Kruisselbrink - [mek@chromium.org](mailto:mek@chromium.org)
 *   Staphany Park - [staphany@chromium.org](mailto:staphany@chromium.org)
 *   Victor Costan - [pwnall@chromium.org](mailto:pwnall@chromium.org)
+*   Ayu Ishii - [ayui@chromium.org](mailto:ayui@chromium.org)
```

Comment:
nit: This list is alphabetically sorted. I think you get to go first! 😛

Reply:
:sweat_smile: :+1: